rfcs/070-concepts-api-changes/README.md

```diff
@@ -0,0 +1,324 @@
+    "Sanitation systems"
+  ]
+    
+  // If a more specific type was available, we would choose it over a less specific type.
+  // For example, sometimes two 'merged' concepts might have types 'Agent' and 'Person'. In this case, we would 
+  // choose 'Person'. 
+  "type": "Concept"
+
+  // The MeSH description takes priority over the Wikidata description.
+  "description": "The development and establishment of environmental conditions favorable to the health of the public.",
+  
+  // Notice that only one Wellcome ID is listed for each linked concept even when multiple 'merged' IDs are available.
+  // We only need to supply one ID (at random), and when the user visits the corresponding theme page, the other 'merged'
+  // IDs will be supplied by Elasticsearch.
+  // In other words, one Wellcome ID should always be enough to uniquely identify a theme page, even if the 
+  // theme page comprises several concepts, each with its own ID.    
+  "narrowerThan": [
+    {
+        "label": "Public Health",
+        "id": "c9ayxjtj"
+    },
+    {
+        "label": "Environmental Health",
+        "id": "sagccnc9" 
+    }
+  ],
+  
+  "relatedTo": [
+    {
+        "label": "Communicable Disease Control",
+        "id": "yz3xs9c9"
+    },
+    {
+        "label": "Hygiene",
+        "id": "sjxv6uys"
+    },
+    {
+        "label": "Sanitary engineering",
+        "id": "zc9y7m45"
+    },
+    {
+        "label": "Environmental policy",
+        "id": "xtndzsda"
+    }
+  ]
+}
+```
+
+## Other considerations
+
+### Wikidata edges to 'concept' node
+
+There are ~200 Wikidata nodes which link to a node labelled `concept` ([Q151885](https://www.wikidata.org/wiki/Q151885)) via the `HAS_PARENT` edge. This node is too
+general, and we should probably exclude it when returning `relatedTo` or `narrowerThan` nodes.
+> <img src="img/has_parent_concept_links.png" alt="has_parent_concept_links" width="600"/>
+
+### Too many source concept edges
+
+Some label-derived concepts are currently linked to multiple source concepts from the same source ontology via `HAS_SOURCE_CONCEPT` edges. 
+For example, the label-derived concept `Cleanliness` is connected to three separate LoC subject headings — `Sanitation`, `Hygiene`, and `Baths`.
+All of these connections were created by matching the label of the label-derived concept to alternative labels of the LoC source concepts.
+This and similar connections can pose several issues:
+  * By linking all of these concepts together, we would be merging the concept pages for 'Sanitation', 'Hygiene', and 'Baths'
+    into a single page. 
+  * In cases where a concept is connected to a given source concept **and** the parent of the source concept, we would end up
+    listing a given concept as a related concept on its own theme page (i.e. a concept would be related to, or part of, itself).
+
+To avoid these issues, we should limit ourselves to creating at most one `HAS_SOURCE_CONCEPT` node from a label-derived concept node.
```

Comment:
I would favour limiting to a most one `HAS_SOURCE_CONCEPT` _within a known ontology_. This is to allow matching across MeSH & LoC, but reduce the risk of a concept being connected to itself through a proliferation of LoC label / alternative label to a label derived concept though a text match.